sky/tools/create_xcframework.py

```diff
@@ -22,15 +22,22 @@ def main():
       help='The framework paths used to create the XCFramework.',
       required=True
   )
+  parser.add_argument(
```

Comment:
It looks like the new argument isn't passed in this PR. What's the plan for using the new code in this file?

Reply:
It's passed from create_full_ios_framework.py, which uses it because the invocation in .ci.yaml already uses the `--dsym` flag.

Reply:
YOU ARE IN A MAZE OF TWISTY PYTHON SCRIPTS, ALL ALIKE. YOU ARE LIKELY TO BE EATEN BY A GRUE.

Reply:
LIGHT LAMP.